.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
please add .vscode to gitignore.
This is a public repo. Even the PRs and comments are public.